build/GridItem.js

```diff
@@ -70,7 +70,7 @@ var GridItem = React.createClass({
     handle: React.PropTypes.string
   },
 
-  getDefaultProps: function () {
+  getDefaultProps: function getDefaultProps() {
```

Comment:
Which version of 6to5 are you using? I wonder why the build came out this way for you; this is definitely better for stack traces.